tests/src/utility/constants.rs

```diff
@@ -1,28 +1,16 @@
-use casper_engine_test_support::{DEFAULT_ACCOUNT_ADDR, DEFAULT_ACCOUNT_PUBLIC_KEY};
-use casper_types::{
-    account::AccountHash, addressable_entity::EntityKindTag, AddressableEntityHash, Key, PublicKey,
-    SecretKey,
-};
+use casper_engine_test_support::DEFAULT_ACCOUNT_ADDR;
+use casper_types::{account::AccountHash, Key, PublicKey, SecretKey};
 use once_cell::sync::Lazy;
 
 pub const BALANCE_OF_SESSION_WASM: &str = "balance_of_call.wasm";
-pub const CONTRACT_1_0_0_WASM: &str = "1_0_0/contract.wasm";
```

Comment:
You can probably update the loop in the make file too that fetches those versions